CommunityToolkit.Mvvm.SourceGenerators/EmbeddedResources/ObservableObject.cs

```diff
@@ -447,7 +450,10 @@ async void MonitorTask()
                 OnPropertyChanged(propertyName);
             }
 
-            callback(newValue);
+            if (callback != null)
```

Comment:
It's the same. I just left this as `!=` to avoid diverging from the reference here:

https://github.com/CommunityToolkit/dotnet/blob/de2355c1c9c937f850f887092f8219f374f9f400/CommunityToolkit.Mvvm/ComponentModel/ObservableObject.cs#L510-L513

This way this file will remain as close as possible, since it's meant to basically be a mirror. Couldn't use `is not null` here because that requires C# 9, and I wanted the generated code to only require C# 8 instead. I've added an explicit cast in 3c9dc47302f82a8bd1eb0888b61da4d1415746f2 just for good measure and to possibly help the codegen, good catch!